Debug error: stack was corrupted when running tests

When I try to run CoreCLR tests, I get a Debug Error indicating that the stack was corrupted. What I did:

0. `git clean -xdf`.
1. `build.cmd`: succeeds.
2. `build-test.cmd`: succeeds.
3. `tests\runtest.cmd` displays the following error dialog:

    ![](https://user-images.githubusercontent.com/287848/58819625-efb9e380-8630-11e9-88fa-cc350d1733cc.png)

Its console output:

```
python "C:\code\coreclr\tests\runtest.py" -arch x64 -build_type Debug --generate_layout
host_os                  :Windows_NT
arch                     :x64
build_type               :Debug
coreclr_repo_location    :C:\code\coreclr
product_location         :C:\code\coreclr\bin\Product\Windows_NT.x64.Debug
core_root                :C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root
test_location            :C:\code\coreclr\bin\tests\Windows_NT.x64.Debug
test_native_bin_location :None
Restoring packages...
C:\code\coreclr\dotnet.cmd msbuild /nologo /verbosity:minimal /clp:Summary "/l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log" /nodeReuse:false /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true /p:UsePartialNGENOptimization=false /maxcpucount C:\code\coreclr\tests\build.proj /fileloggerparameters:"Verbosity=normal;LogFile=C:\code\coreclr\bin\Logs\Restore_ProductWindows_NT_x64_Debug.log" /fileloggerparameters1:"WarningsOnly;LogFile=C:\code\coreclr\bin\Logs\Restore_ProductWindows_NT_x64_Debug.wrn" /fileloggerparameters2:"ErrorsOnly;LogFile=C:\code\coreclr\bin\Logs\Restore_ProductWindows_NT_x64_Debug.err" /t:BatchRestorePackages /p:__BuildType=Debug /p:__BuildArch=x64 /p:__BuildOS=Windows_NT

Creating Core_Root...
C:\code\coreclr\dotnet.cmd msbuild /nologo /verbosity:minimal /clp:Summary "/l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log" /nodeReuse:false /p:RestoreDefaultOptimizationDataPackage=false /p:PortableBuild=true /p:UsePartialNGENOptimization=false /maxcpucount C:\code\coreclr\tests\runtest.proj /fileloggerparameters:"Verbosity=normal;LogFile=C:\code\coreclr\bin\Logs\Tests_Overlay_ManagedWindows_NT_x64_Debug.log" /fileloggerparameters1:"WarningsOnly;LogFile=C:\code\coreclr\bin\Logs\Tests_Overlay_ManagedWindows_NT_x64_Debug.wrn" /fileloggerparameters2:"ErrorsOnly;LogFile=C:\code\coreclr\bin\Logs\Tests_Overlay_ManagedWindows_NT_x64_Debug.err" /t:CreateTestOverlay /p:__BuildType=Debug /p:__BuildArch=x64 /p:__BuildOS=Windows_NT

Copying Product Bin to Core_Root:
cp -r C:\code\coreclr\bin\Product\Windows_NT.x64.Debug\* C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root
---------------------------------------------------------------------

Core_Root setup.

Setting __TestTimeout=900000
Setting CORE_ROOT=C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root
Download and overwrite xunit.console.dll in Core_Root
C:\code\coreclr\dotnet.cmd msbuild C:\code\coreclr\tests\runtest.proj /p:Runtests=true /clp:showcommandline /fileloggerparameters:"Verbosity=normal;LogFile=C:\code\coreclr\bin\Logs\TestRunResults_Windows_NT_x64_Debug.log" /fileloggerparameters1:"WarningsOnly;LogFile=C:\code\coreclr\bin\Logs\TestRunResults_Windows_NT_x64_Debug.wrn" /fileloggerparameters2:"ErrorsOnly;LogFile=C:\code\coreclr\bin\Logs\TestRunResults_Windows_NT_x64_Debug.err" /consoleloggerparameters:Summary /p:__BuildOS=Windows_NT /p:__BuildArch=x64 /p:__BuildType=Debug /p:__LogsDir=C:\code\coreclr\bin\Logs
Installing dotnet using Arcade...
running: powershell -NoProfile -ExecutionPolicy unrestricted -Command ". C:\code\coreclr\eng\configure-toolset.ps1; . C:\code\coreclr\eng\common\tools.ps1; InitializeBuildTool"

Name                           Value
----                           -----
Path                           C:\code\coreclr\.dotnet\dotnet.exe
Command                        msbuild
Tool                           dotnet
Framework                      netcoreapp2.1


Running: dotnet msbuild C:\code\coreclr\tests\runtest.proj /p:Runtests=true /clp:showcommandline /fileloggerparameters:\"Verbosity=normal;LogFile=C:\code\coreclr\bin\Logs\TestRunResults_Windows_NT_x64_Debug.log\" /fileloggerparameters1:\"WarningsOnly;LogFile=C:\code\coreclr\bin\Logs\TestRunResults_Windows_NT_x64_Debug.wrn\" /fileloggerparameters2:\"ErrorsOnly;LogFile=C:\code\coreclr\bin\Logs\TestRunResults_Windows_NT_x64_Debug.err\" /consoleloggerparameters:Summary /p:__BuildOS=Windows_NT /p:__BuildArch=x64 /p:__BuildType=Debug /p:__LogsDir=C:\code\coreclr\bin\Logs
Microsoft (R) Build Engine version 16.1.67-preview+g13843078ee for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root\corerun.exe C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root\xunit.console.dll .\baseservices\compilerservices\baseservices.compilerservices.XUnitWrapper.dll .\baseservices\threading\baseservices.threading.XUnitWrapper.dll .\baseservices\TieredCompilation\baseservices.TieredCompilation.XUnitWrapper.dll .\baseservices\typeequivalence\baseservices.typeequivalence.XUnitWrapper.dll .\baseservices\varargs\baseservices.varargs.XUnitWrapper.dll .\CoreMangLib\system\CoreMangLib.system.XUnitWrapper.dll .\Exceptions\Finalization\Exceptions.Finalization.XUnitWrapper.dll .\Exceptions\ForeignThread\Exceptions.ForeignThread.XUnitWrapper.dll .\GC\API\GC.API.XUnitWrapper.dll .\GC\Coverage\GC.Coverage.XUnitWrapper.dll .\GC\Features\GC.Features.XUnitWrapper.dll .\GC\LargeMemory\GC.LargeMemory.XUnitWrapper.dll .\GC\Regressions\GC.Regressions.XUnitWrapper.dll .\GC\Scenarios\GC.Scenarios.XUnitWrapper.dll .\ilasm\System\ilasm.System.XUnitWrapper.dll .\Interop\ArrayMarshalling\Interop.ArrayMarshalling.XUnitWrapper.dll .\Interop\COM\Interop.COM.XUnitWrapper.dll .\Interop\ExecInDefAppDom\Interop.ExecInDefAppDom.XUnitWrapper.dll .\Interop\ICastable\Interop.ICastable.XUnitWrapper.dll .\Interop\ICustomMarshaler\Interop.ICustomMarshaler.XUnitWrapper.dll .\Interop\IJW\Interop.IJW.XUnitWrapper.dll .\Interop\LayoutClass\Interop.LayoutClass.XUnitWrapper.dll .\Interop\MarshalAPI\Interop.MarshalAPI.XUnitWrapper.dll .\Interop\NativeLibraryResolveCallback\Interop.NativeLibraryResolveCallback.XUnitWrapper.dll .\Interop\NativeLibraryResolveEvent\Interop.NativeLibraryResolveEvent.XUnitWrapper.dll .\Interop\NativeLibrary\Interop.NativeLibrary.XUnitWrapper.dll .\Interop\PInvoke\Interop.PInvoke.XUnitWrapper.dll .\Interop\StringMarshalling\Interop.StringMarshalling.XUnitWrapper.dll .\Interop\StructMarshalling\Interop.StructMarshalling.XUnitWrapper.dll .\Interop\StructPacking\Interop.StructPacking.XUnitWrapper.dll .\Interop\WinRT\Interop.WinRT.XUnitWrapper.dll .\JIT\CheckProjects\JIT.CheckProjects.XUnitWrapper.dll .\JIT\CodeGenBringUpTests\JIT.CodeGenBringUpTests.XUnitWrapper.dll .\JIT\Directed\JIT.Directed.XUnitWrapper.dll .\JIT\Generics\JIT.Generics.XUnitWrapper.dll .\JIT\HardwareIntrinsics\JIT.HardwareIntrinsics.XUnitWrapper.dll .\JIT\IL_Conformance\JIT.IL_Conformance.XUnitWrapper.dll .\JIT\Intrinsics\JIT.Intrinsics.XUnitWrapper.dll .\JIT\jit64\JIT.jit64.XUnitWrapper.dll .\JIT\Methodical\JIT.Methodical.XUnitWrapper.dll .\JIT\opt\JIT.opt.XUnitWrapper.dll .\JIT\Performance\JIT.Performance.XUnitWrapper.dll .\JIT\Regression\JIT.Regression.XUnitWrapper.dll .\JIT\RyuJIT\JIT.RyuJIT.XUnitWrapper.dll .\JIT\SIMD\JIT.SIMD.XUnitWrapper.dll .\JIT\superpmi\JIT.superpmi.XUnitWrapper.dll .\Loader\AssemblyLoadContext30Extensions\Loader.AssemblyLoadContext30Extensions.XUnitWrapper.dll .\Loader\classloader\Loader.classloader.XUnitWrapper.dll .\Loader\ContextualReflection\Loader.ContextualReflection.XUnitWrapper.dll .\Loader\regressions\Loader.regressions.XUnitWrapper.dll .\managed\Compilation\managed.Compilation.XUnitWrapper.dll .\readytorun\DynamicMethodGCStress\readytorun.DynamicMethodGCStress.XUnitWrapper.dll .\readytorun\r2rdump\readytorun.r2rdump.XUnitWrapper.dll .\readytorun\tests\readytorun.tests.XUnitWrapper.dll .\reflection\DefaultInterfaceMethods\reflection.DefaultInterfaceMethods.XUnitWrapper.dll .\reflection\Modifiers\reflection.Modifiers.XUnitWrapper.dll .\reflection\SetValue\reflection.SetValue.XUnitWrapper.dll .\reflection\StaticInterfaceMembers\reflection.StaticInterfaceMembers.XUnitWrapper.dll .\Regressions\common\Regressions.common.XUnitWrapper.dll .\Regressions\coreclr\Regressions.coreclr.XUnitWrapper.dll .\tracing\eventactivityidcontrol\tracing.eventactivityidcontrol.XUnitWrapper.dll .\tracing\eventcounter\tracing.eventcounter.XUnitWrapper.dll .\tracing\eventlistener\tracing.eventlistener.XUnitWrapper.dll .\tracing\eventsource\tracing.eventsource.XUnitWrapper.dll .\tracing\keyword\tracing.keyword.XUnitWrapper.dll .\tracing\regress\tracing.regress.XUnitWrapper.dll .\tracing\tracecontrol\tracing.tracecontrol.XUnitWrapper.dll .\tracing\tracevalidation\tracing.tracevalidation.XUnitWrapper.dll -parallel collections -html C:\code\coreclr\bin\Logs\TestRun.html -xml C:\code\coreclr\bin\Logs\TestRun.xml  -notrait category=outerloop -notrait category=failing -nocolor
  xUnit.net Console Runner v2.4.1 (64-bit .NET Core 3.0.0-dev.19303.1)
    Discovering: baseservices.compilerservices.XUnitWrapper
    Discovered:  baseservices.compilerservices.XUnitWrapper
    Starting:    baseservices.compilerservices.XUnitWrapper
C:\code\coreclr\tests\tests.targets(73,5): error MSB3073: The command "C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root\corerun.exe C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root\xunit.console.dll .\baseservices\compilerservices\baseservices.compilerservices.XUnitWrapper.dll .\baseservices\threading\baseservices.threading.XUnitWrapper.dll .\baseservices\TieredCompilation\baseservices.TieredCompilation.XUnitWrapper.dll .\baseservices\typeequivalence\baseservices.typeequivalence.XUnitWrapper.dll .\baseservices\varargs\baseservices.varargs.XUnitWrapper.dll .\CoreMangLib\system\CoreMangLib.system.XUnitWrapper.dll .\Exceptions\Finalization\Exceptions.Finalization.XUnitWrapper.dll .\Exceptions\ForeignThread\Exceptions.ForeignThread.XUnitWrapper.dll .\GC\API\GC.API.XUnitWrapper.dll .\GC\Coverage\GC.Coverage.XUnitWrapper.dll .\GC\Features\GC.Features.XUnitWrapper.dll .\GC\LargeMemory\GC.LargeMemory.XUnitWrapper.dll .\GC\Regressions\GC.Regressions.XUnitWrapper.dll .\GC\Scenarios\GC.Scenarios.XUnitWrapper.dll .\ilasm\System\ilasm.System.XUnitWrapper.dll .\Interop\ArrayMarshalling\Interop.ArrayMarshalling.XUnitWrapper.dll .\Interop\COM\Interop.COM.XUnitWrapper.dll .\Interop\ExecInDefAppDom\Interop.ExecInDefAppDom.XUnitWrapper.dll .\Interop\ICastable\Interop.ICastable.XUnitWrapper.dll .\Interop\ICustomMarshaler\Interop.ICustomMarshaler.XUnitWrapper.dll .\Interop\IJW\Interop.IJW.XUnitWrapper.dll .\Interop\LayoutClass\Interop.LayoutClass.XUnitWrapper.dll .\Interop\MarshalAPI\Interop.MarshalAPI.XUnitWrapper.dll .\Interop\NativeLibraryResolveCallback\Interop.NativeLibraryResolveCallback.XUnitWrapper.dll .\Interop\NativeLibraryResolveEvent\Interop.NativeLibraryResolveEvent.XUnitWrapper.dll .\Interop\NativeLibrary\Interop.NativeLibrary.XUnitWrapper.dll .\Interop\PInvoke\Interop.PInvoke.XUnitWrapper.dll .\Interop\StringMarshalling\Interop.StringMarshalling.XUnitWrapper.dll .\Interop\StructMarshalling\Interop.StructMarshalling.XUnitWrapper.dll .\Interop\StructPacking\Interop.StructPacking.XUnitWrapper.dll .\Interop\WinRT\Interop.WinRT.XUnitWrapper.dll .\JIT\CheckProjects\JIT.CheckProjects.XUnitWrapper.dll .\JIT\CodeGenBringUpTests\JIT.CodeGenBringUpTests.XUnitWrapper.dll .\JIT\Directed\JIT.Directed.XUnitWrapper.dll .\JIT\Generics\JIT.Generics.XUnitWrapper.dll .\JIT\HardwareIntrinsics\JIT.HardwareIntrinsics.XUnitWrapper.dll .\JIT\IL_Conformance\JIT.IL_Conformance.XUnitWrapper.dll .\JIT\Intrinsics\JIT.Intrinsics.XUnitWrapper.dll .\JIT\jit64\JIT.jit64.XUnitWrapper.dll .\JIT\Methodical\JIT.Methodical.XUnitWrapper.dll .\JIT\opt\JIT.opt.XUnitWrapper.dll .\JIT\Performance\JIT.Performance.XUnitWrapper.dll .\JIT\Regression\JIT.Regression.XUnitWrapper.dll .\JIT\RyuJIT\JIT.RyuJIT.XUnitWrapper.dll .\JIT\SIMD\JIT.SIMD.XUnitWrapper.dll .\JIT\superpmi\JIT.superpmi.XUnitWrapper.dll .\Loader\AssemblyLoadContext30Extensions\Loader.AssemblyLoadContext30Extensions.XUnitWrapper.dll .\Loader\classloader\Loader.classloader.XUnitWrapper.dll .\Loader\ContextualReflection\Loader.ContextualReflection.XUnitWrapper.dll .\Loader\regressions\Loader.regressions.XUnitWrapper.dll .\managed\Compilation\managed.Compilation.XUnitWrapper.dll .\readytorun\DynamicMethodGCStress\readytorun.DynamicMethodGCStress.XUnitWrapper.dll .\readytorun\r2rdump\readytorun.r2rdump.XUnitWrapper.dll .\readytorun\tests\readytorun.tests.XUnitWrapper.dll .\reflection\DefaultInterfaceMethods\reflection.DefaultInterfaceMethods.XUnitWrapper.dll .\reflection\Modifiers\reflection.Modifiers.XUnitWrapper.dll .\reflection\SetValue\reflection.SetValue.XUnitWrapper.dll .\reflection\StaticInterfaceMembers\reflection.StaticInterfaceMembers.XUnitWrapper.dll .\Regressions\common\Regressions.common.XUnitWrapper.dll .\Regressions\coreclr\Regressions.coreclr.XUnitWrapper.dll .\tracing\eventactivityidcontrol\tracing.eventactivityidcontrol.XUnitWrapper.dll .\tracing\eventcounter\tracing.eventcounter.XUnitWrapper.dll .\tracing\eventlistener\tracing.eventlistener.XUnitWrapper.dll .\tracing\eventsource\tracing.eventsource.XUnitWrapper.dll .\tracing\keyword\tracing.keyword.XUnitWrapper.dll .\tracing\regress\tracing.regress.XUnitWrapper.dll .\tracing\tracecontrol\tracing.tracecontrol.XUnitWrapper.dll .\tracing\tracevalidation\tracing.tracevalidation.XUnitWrapper.dll -parallel collections -html C:\code\coreclr\bin\Logs\TestRun.html -xml C:\code\coreclr\bin\Logs\TestRun.xml  -notrait category=outerloop -notrait category=failing -nocolor" exited with code 3. [C:\code\coreclr\tests\runtest.proj]

Build FAILED.

C:\code\coreclr\tests\tests.targets(73,5): error MSB3073: The command "C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root\corerun.exe C:\code\coreclr\bin\tests\Windows_NT.x64.Debug\Tests\Core_Root\xunit.console.dll .\baseservices\compilerservices\baseservices.compilerservices.XUnitWrapper.dll .\baseservices\threading\baseservices.threading.XUnitWrapper.dll .\baseservices\TieredCompilation\baseservices.TieredCompilation.XUnitWrapper.dll .\baseservices\typeequivalence\baseservices.typeequivalence.XUnitWrapper.dll .\baseservices\varargs\baseservices.varargs.XUnitWrapper.dll .\CoreMangLib\system\CoreMangLib.system.XUnitWrapper.dll .\Exceptions\Finalization\Exceptions.Finalization.XUnitWrapper.dll .\Exceptions\ForeignThread\Exceptions.ForeignThread.XUnitWrapper.dll .\GC\API\GC.API.XUnitWrapper.dll .\GC\Coverage\GC.Coverage.XUnitWrapper.dll .\GC\Features\GC.Features.XUnitWrapper.dll .\GC\LargeMemory\GC.LargeMemory.XUnitWrapper.dll .\GC\Regressions\GC.Regressions.XUnitWrapper.dll .\GC\Scenarios\GC.Scenarios.XUnitWrapper.dll .\ilasm\System\ilasm.System.XUnitWrapper.dll .\Interop\ArrayMarshalling\Interop.ArrayMarshalling.XUnitWrapper.dll .\Interop\COM\Interop.COM.XUnitWrapper.dll .\Interop\ExecInDefAppDom\Interop.ExecInDefAppDom.XUnitWrapper.dll .\Interop\ICastable\Interop.ICastable.XUnitWrapper.dll .\Interop\ICustomMarshaler\Interop.ICustomMarshaler.XUnitWrapper.dll .\Interop\IJW\Interop.IJW.XUnitWrapper.dll .\Interop\LayoutClass\Interop.LayoutClass.XUnitWrapper.dll .\Interop\MarshalAPI\Interop.MarshalAPI.XUnitWrapper.dll .\Interop\NativeLibraryResolveCallback\Interop.NativeLibraryResolveCallback.XUnitWrapper.dll .\Interop\NativeLibraryResolveEvent\Interop.NativeLibraryResolveEvent.XUnitWrapper.dll .\Interop\NativeLibrary\Interop.NativeLibrary.XUnitWrapper.dll .\Interop\PInvoke\Interop.PInvoke.XUnitWrapper.dll .\Interop\StringMarshalling\Interop.StringMarshalling.XUnitWrapper.dll .\Interop\StructMarshalling\Interop.StructMarshalling.XUnitWrapper.dll .\Interop\StructPacking\Interop.StructPacking.XUnitWrapper.dll .\Interop\WinRT\Interop.WinRT.XUnitWrapper.dll .\JIT\CheckProjects\JIT.CheckProjects.XUnitWrapper.dll .\JIT\CodeGenBringUpTests\JIT.CodeGenBringUpTests.XUnitWrapper.dll .\JIT\Directed\JIT.Directed.XUnitWrapper.dll .\JIT\Generics\JIT.Generics.XUnitWrapper.dll .\JIT\HardwareIntrinsics\JIT.HardwareIntrinsics.XUnitWrapper.dll .\JIT\IL_Conformance\JIT.IL_Conformance.XUnitWrapper.dll .\JIT\Intrinsics\JIT.Intrinsics.XUnitWrapper.dll .\JIT\jit64\JIT.jit64.XUnitWrapper.dll .\JIT\Methodical\JIT.Methodical.XUnitWrapper.dll .\JIT\opt\JIT.opt.XUnitWrapper.dll .\JIT\Performance\JIT.Performance.XUnitWrapper.dll .\JIT\Regression\JIT.Regression.XUnitWrapper.dll .\JIT\RyuJIT\JIT.RyuJIT.XUnitWrapper.dll .\JIT\SIMD\JIT.SIMD.XUnitWrapper.dll .\JIT\superpmi\JIT.superpmi.XUnitWrapper.dll .\Loader\AssemblyLoadContext30Extensions\Loader.AssemblyLoadContext30Extensions.XUnitWrapper.dll .\Loader\classloader\Loader.classloader.XUnitWrapper.dll .\Loader\ContextualReflection\Loader.ContextualReflection.XUnitWrapper.dll .\Loader\regressions\Loader.regressions.XUnitWrapper.dll .\managed\Compilation\managed.Compilation.XUnitWrapper.dll .\readytorun\DynamicMethodGCStress\readytorun.DynamicMethodGCStress.XUnitWrapper.dll .\readytorun\r2rdump\readytorun.r2rdump.XUnitWrapper.dll .\readytorun\tests\readytorun.tests.XUnitWrapper.dll .\reflection\DefaultInterfaceMethods\reflection.DefaultInterfaceMethods.XUnitWrapper.dll .\reflection\Modifiers\reflection.Modifiers.XUnitWrapper.dll .\reflection\SetValue\reflection.SetValue.XUnitWrapper.dll .\reflection\StaticInterfaceMembers\reflection.StaticInterfaceMembers.XUnitWrapper.dll .\Regressions\common\Regressions.common.XUnitWrapper.dll .\Regressions\coreclr\Regressions.coreclr.XUnitWrapper.dll .\tracing\eventactivityidcontrol\tracing.eventactivityidcontrol.XUnitWrapper.dll .\tracing\eventcounter\tracing.eventcounter.XUnitWrapper.dll .\tracing\eventlistener\tracing.eventlistener.XUnitWrapper.dll .\tracing\eventsource\tracing.eventsource.XUnitWrapper.dll .\tracing\keyword\tracing.keyword.XUnitWrapper.dll .\tracing\regress\tracing.regress.XUnitWrapper.dll .\tracing\tracecontrol\tracing.tracecontrol.XUnitWrapper.dll .\tracing\tracevalidation\tracing.tracevalidation.XUnitWrapper.dll -parallel collections -html C:\code\coreclr\bin\Logs\TestRun.html -xml C:\code\coreclr\bin\Logs\TestRun.xml  -notrait category=outerloop -notrait category=failing -nocolor" exited with code 3. [C:\code\coreclr\tests\runtest.proj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:09:59.33
Test run finished.
Parsing test results from (C:\code\coreclr\bin\Logs\TestRunResults_Windows_NT_x64_Debug)
Unable to find testRun.xml. This normally means the tests did not run.
It could also mean there was a problem logging. Please run the tests again.
```

If there is something I can do to help diagnose this issue, please tell me.

--- a/src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs
+++ b/src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs
@@ -673,7 +673,7 @@ namespace System.Reflection.Emit
             CanSkipCSEvaluation = 0x8,
         }
 
-        internal override RuntimeType? GetJitContext(ref int securityControlFlags)
+        internal override RuntimeType? GetJitContext(out int securityControlFlags)
         {
             RuntimeType? typeOwner;
 
@@ -703,7 +703,7 @@ namespace System.Reflection.Emit
         }
 
         internal override byte[] GetCodeInfo(
-            ref int stackSize, ref int initLocals, ref int EHCount)
+            out int stackSize, out int initLocals, out int EHCount)
         {
             stackSize = m_stackSize;
             if (m_exceptionHeader != null && m_exceptionHeader.Length != 0)

--- a/src/System.Private.CoreLib/src/System/RuntimeHandles.cs
+++ b/src/System.Private.CoreLib/src/System/RuntimeHandles.cs
@@ -1545,8 +1545,8 @@ namespace System
         }
 
         // ILHeader info
-        internal abstract RuntimeType GetJitContext(ref int securityControlFlags);
-        internal abstract byte[] GetCodeInfo(ref int stackSize, ref int initLocals, ref int EHCount);
+        internal abstract RuntimeType GetJitContext(out int securityControlFlags);
+        internal abstract byte[] GetCodeInfo(out int stackSize, out int initLocals, out int EHCount);
         internal abstract byte[] GetLocalsSignature();
         internal abstract unsafe void GetEHInfo(int EHNumber, void* exception);
         internal abstract byte[] GetRawEHInfo();

--- a/src/vm/dynamicmethod.cpp
+++ b/src/vm/dynamicmethod.cpp
@@ -1044,20 +1044,6 @@ void LCGMethodResolver::GetJitContext(SecurityControlFlags * securityControlFlag
     } CONTRACTL_END;
     
     GCX_COOP();
-    GetJitContextCoop(securityControlFlags, typeOwner);
-}
-
-void LCGMethodResolver::GetJitContextCoop(SecurityControlFlags * securityControlFlags,
-                                      TypeHandle *typeOwner)
-{
-    CONTRACTL {
-        THROWS;
-        GC_TRIGGERS;
-        MODE_COOPERATIVE;
-        INJECT_FAULT(COMPlusThrowOM(););
-        PRECONDITION(CheckPointer(securityControlFlags));
-        PRECONDITION(CheckPointer(typeOwner));
-    } CONTRACTL_END;
 
     MethodDescCallSite getJitContext(METHOD__RESOLVER__GET_JIT_CONTEXT, m_managedResolver);
 
@@ -1098,8 +1084,7 @@ BYTE* LCGMethodResolver::GetCodeInfo(unsigned *pCodeSize, unsigned *pStackSize, 
         OBJECTREF resolver = ObjectFromHandle(m_managedResolver);
         VALIDATEOBJECTREF(resolver); // gc root must be up the stack
 
-        DWORD initLocals = 0, EHSize = 0;
-        unsigned short stackSize = 0;
+        int32_t stackSize = 0, initLocals = 0, EHSize = 0;
         ARG_SLOT args[] =
         {
             ObjToArgSlot(resolver),

--- a/src/vm/dynamicmethod.h
+++ b/src/vm/dynamicmethod.h
@@ -115,8 +115,6 @@ public:
     void FreeCompileTimeState();
     void GetJitContext(SecurityControlFlags * securityControlFlags,
                        TypeHandle * typeOwner);
-    void GetJitContextCoop(SecurityControlFlags * securityControlFlags,
-                           TypeHandle * typeOwner);
     ChunkAllocator* GetJitMetaHeap();
 
     BYTE* GetCodeInfo(unsigned *pCodeSize, unsigned *pStackSize, CorInfoOptions *pOptions, unsigned* pEHSize);

